### PR TITLE
a few web tweaks for a11y assessment app

### PR DESCRIPTION
**Pull request by yjbanov**
Mostly tweaks for better focus management, namely:

* Use `autofocus` throughout so the a11y focus is transferred to a logical place when overlaid content pops up (screen transitions, dialogs).
* Consolidate "enabled" and "disabled" widgets into the same screen. Otherwise, when only a disabled widget is shown, there's nothing to focus on and the screen reader is lost.

### CODE REVIEW

File: dev/a11y_assessments/lib/use_cases/check_box_list_tile.dart [hunk 1]
```diff
@@ -30,16 +30,30 @@ class _MainWidgetState extends State<_MainWidget> {
...
+            value: _checked,
+            onChanged: (bool? value) {
+              setState(() {
+                _checked = value!;
+              });
+            },
+            title: const Text('a check box list title'),
+          ),
+          CheckboxListTile(
+            autofocus: true,
```
Comment (chunhtai) [inline dev/a11y_assessments/lib/use_cases/check_box_list_tile.dart:46]: This is not needed?

File: dev/a11y_assessments/lib/use_cases/check_box_list_tile.dart [hunk 2]
```diff
@@ -30,16 +30,30 @@ class _MainWidgetState extends State<_MainWidget> {
...
-            setState(() {
-              _checked = value!;
-            });
-          },
-          title: const Text('a check box list title'),
-        ),
+      body: ListView(
+        children: <Widget>[
+          CheckboxListTile(
+            autofocus: true,
```
Comment (chunhtai) [inline dev/a11y_assessments/lib/use_cases/check_box_list_tile.dart:36]: Do you have an open issue for fixing the focus? if so we should add a todo here.

**Review by chunhtai — APPROVED** [review]
LGTM

File: dev/a11y_assessments/lib/use_cases/check_box_list_tile.dart (hunk 1 above)
Comment (yjbanov) [reply dev/a11y_assessments/lib/use_cases/check_box_list_tile.dart:46]: Oops! Removed.

File: dev/a11y_assessments/lib/use_cases/check_box_list_tile.dart (hunk 2 above)
Comment (yjbanov) [reply dev/a11y_assessments/lib/use_cases/check_box_list_tile.dart:36]: Left a TODO in `main.dart`, since I'm using `autofocus` a lot in the app.

**Pull request merged**